DEVELOPMENT.md

```diff
@@ -79,6 +81,7 @@
     If you run into an error: `Error: Call to undefined function Google\Protobuf\Internal\bccomp()`, that is because the [BC Math](https://www.php.net/manual/en/book.bc.php) extension is not always included by default (see tracking bug here: https://github.com/protocolbuffers/protobuf/issues/4465). You can get around this by installing BC Math with the command `sudo apt install php-bcmath`.
 
     Updating unit test goldens:
+    > This is useful if you have changed something in the generation process(_for example, renamed a variable in a generator_) and you want to recreate the test files that will be used for comparison.
```

Comment:
nit: just make this text instead of a quote. Also, I changed the verbage slightly
```suggestion
    You will need to update the golden test files if you change something in the generation process that modifies the output (_for example, renaming a variable).
```